src/graph/svg-renderer.ts

```diff
@@ -2,7 +2,8 @@ import { getDotSelector } from './dot';
 import { observeStore } from '../redux';
 import { svgRenderingFinished, reportError } from '../actions';
 
-import { loadWorker } from '../utils/';
+import { loadWorker as internalLoadWorker } from '../utils/';
```

Comment:
I think `defaultLoadWorker` is better name

Reply:
Agreed.